### ISSUE TITLE
What about Android Q?

### ISSUE THREAD

**Issue by gigby** [devstepbcn/react-native-android-wifi#90]
According to [this](https://developer.android.com/preview/privacy/camera-connectivity) document, API of this package should be changed?

> Restriction on enabling and disabling Wi-Fi
> Apps targeting Android Q cannot enable or disable Wi-Fi. The WifiManager.setWifiEnabled() method always returns false.
> 
> If needed, use a settings panel to prompt users to enable and disable Wi-Fi.

**Comment by metahexane** [comment]
This needs more attention. Library becomes unusable for newer devices.

**Comment by VSV6** [comment]
Yes, in android 11 this library doesn't work as expected. When this problem gets solved?
